kubernetes.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: amida-auth-microservice-deployment
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: amida-auth-microservice
+    spec:
+      imagePullSecrets:
+      - name: regsecret
+      containers:
+      - name: amida-auth-microservice
+        image: amidatech/auth-service
+        env:
+        - name: PG_DB
+          value: amida_auth
+        - name: PG_PORT
+          value: '5432'
+        - name: PG_HOST
+          value: kubernetes-auth-microservice.czgzedfwgy7z.us-west-2.rds.amazonaws.com
```

Comment:
Same as the other repo--we should make this a command line param, rather than hard-coding RDS info that will end up in github.

Reply:
Yes @jsachs . Good catch. The dbs are locked down in a private subnet. But was going to be turning these over regardless.